CMakeLists.txt

```diff
@@ -23,7 +23,9 @@ include(GNUInstallDirs)
 
 # Set when installed via make install
 set(open62541_TOOLS_DIR ${PROJECT_SOURCE_DIR}/tools)
-set(open62541_NODESET_DIR ${PROJECT_SOURCE_DIR}/deps/ua-nodeset)
+if(NOT UA_NODESET_DIR)
```

Comment:
If you use `option(UA_NODESET_DIR ...)` then the configuration appears in the GUI tools for cmake as well.

Reply:
Good point!
I have changed the code line.